### PR TITLE
Include a log when creating a new labels about what those labels are

### PR DESCRIPTION
**Pull request by orta**
# What Changed

```sh
$ yarn auto create-labels
$
```

`->`

```sh
$ yarn auto create-labels
Created labels: [x, y, z]
$
```

# Why

I didn't know what the labels would be, and even after creating them I didn't.

I couldn't find any other uses of the `logger.log` which makes me think this is the wrong way to do it?

Todo:

- [x] Add tests
- [x] Add docs (n/a)
- [x] Add SemVer label

### CODE REVIEW

**Comment by orta** [comment]
Alternative take: also print out a  url like: `https://github.com/omakase-js/omakase/labels` so that people can inspect?

**Comment by hipstersmoothie** [comment]
Awesome! I like the idea of printing the URL also. I'll merge once you add that 👍 

`logger.log` is fine, I think we just hadn't found an area to use it where it made sense. Thanks for all the contributions!

File: src/git.ts [hunk 1]
```diff
@@ -80,11 +80,21 @@ export default class GitHub {
...
     });
 
-    this.logger.veryVerbose.info('Sucessfully authenticated with GitHub.');
+    this.logger.veryVerbose.info('Successfully authenticated with GitHub.');
 
     return Promise.resolve();
   }
 
+  // Looks like: https://developer.github.com/v3/repos/#get
+  public async getRepoMetadata(): Promise<any> {
```
Comment (hipstersmoothie) [inline src/git.ts]: it looks like `this.ghub.repos.get` return a type of `GHub.Response<GHub.ReposGetResponse>` so the return type for this function should probably be `Promise<GHub.Response<GHub.ReposGetResponse>>`

Comment (hipstersmoothie) [reply src/git.ts]:
```suggestion
  public async getRepoMetadata(): Promise<GHub.Response<GHub.ReposGetResponse>> {
```

**Pull request merged**